addons/sale/i18n/sale.pot

```diff
@@ -3717,7 +3717,7 @@ msgstr ""
 
 #. module: sale
 #: model:ir.actions.act_window,name:sale.action_order_report_products
-msgid "Sales Analysis Products"
+msgid "Sales Analysis By Products"
```

Comment:
Thank you for trying to keep the pot file up to date, but in the future please do not manually update the pot file. Export the file and paste it into the existing one and check the diff for what corresponds to your changes. When you manually update it like this, it may be incorrect and/or cause issues if done in certain ways.

Reply:
What was the incorrect change here ?
AFAICS, the diff is the expected one :thinking:

Reply:
It’s best to keep it in alphabetical order, otherwise it’s too easy to create a duplicated entry (which would make an invalid pot refused by transifex). But actually 47ece9ec71e172cfe9cdf3490a7e211917b60688 was already introducing a bad order.

Reply:
It's not a big deal in this case though since I just re-exported all the pot files. It's mostly for future reference

Reply:
Oh yeah indeed, forgot once again this ordering logic, my bad :man_facepalming: